resources/views/components/radio.blade.php

```diff
@@ -1,6 +1,6 @@
 <label class="flex items-center">
-    <input type="radio" {{ $attributes->merge(['class' => 'focus:ring-neutral h-4 w-4 text-neutral border']) }}>
-    <div class="ml-2 text-gray-700">
+    <input type="radio" {{ $attributes->merge(['class' => 'focus:ring-border-default size-4 text border']) }}>
```

Comment:
ring-border-default?

Reply:
This will be removed and https://github.com/rapidez/blade-components will be used, the files: https://github.com/rapidez/blade-components/tree/master/resources/views/components/input/radio, please check those; if anything needs to change it needs to change there.